### PR TITLE
Add Arduino icon (simple-icons#1359)

### PR DESCRIPTION
**Pull request by adrianoluengo**
**Issue:**

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
Anything relevant, for example:
  - Why did you pick the hex value?
    The color represents the color pallet of the Arduino site, IDE, etc.
  - Did you manually vectorize the logo?
    No
  - Have you used multiple sources?
    No

### CODE REVIEW

File: icons/arduino-white.svg [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
...1.61208 48.46154,-48.46154c0,-26.84946 -21.61208,-48.46154 -48.46154,-48.46154c-19.52975,0 -35.33654,11.61058 -46.1899,22.46394c-4.19621,4.19621 -7.60366,8.23467 -10.34856,11.86298c-2.71334,-3.59675 -5.96304,-7.69832 -10.09615,-11.86298c-10.79026,-10.88491 -26.59706,-22.46394 -46.44231,-22.46394zM48.46154,72.69231c13.28275,0 25.61899,8.61328 34.83173,17.92067c7.22506,7.31971 9.84375,11.61058 11.61058,14.38702c-1.82992,2.83955 -4.57482,7.09886 -11.86298,14.38702c-9.33894,9.33894 -21.80138,17.92067 -34.57933,17.92067c-18.39393,0 -32.30769,-13.91376 -32.30769,-32.30769c0,-18.39393 13.91376,-32.30769 32.30769,-32.30769zM161.53846,72.69231c18.39393,0 32.30769,13.91376 32.30769,32.30769c0,18.39393 -13.91376,32.30769 -32.30769,32.30769c-12.77794,0 -25.24038,-8.58173 -34.57933,-17.92067c-7.28816,-7.28816 -10.03305,-11.54747 -11.86298,-14.38702c1.82993,-2.83954 4.57482,-7.09886 11.86298,-14.38702c9.33894,-9.33894 21.80138,-17.92067 34.57933,-17.92067zM155.98558,88.84615v10.60096h-10.60096v11.10577h10.60096v10.60096h11.10577v-10.60096h10.60096v-11.10577h-10.60096v-10.60096zM32.30769,99.44712v10.34856h32.30769v-10.34856z"/></g></g><path d="" fill="none"/><path d="" fill="none"/></g></g></svg>
```
Comment (hylickipiotr) [inline icons/arduino-white.svg]: Looking at the svg code, it seems you've not optimized it, could you fix that? 🤔

Comment (hylickipiotr) [inline icons/arduino-white.svg]: Can you try to compound the paths in this SVG? If you need help with this let me know 😄

Comment (hylickipiotr) [inline icons/arduino-white.svg]: Can you try to remove unnecessary arttributes in this SVG? 🤔 
Your file should look like this:
`<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>[icon name] icon</title><path d="..."/></svg>`

**Comment by davidklebanoff** [comment]
Hi @adrianoluengo,

Thanks for the contribution! As @hylickipiotr has pointed out, there are a number of issues with the icone submitted. I think it might be a good idea to read the [Contribution Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md) to see the process of adding a new icon to Simple Icons.

**Comment by adrianoluengo** [comment]
I would love to manually fix all the issues with the diff I submitted. As
you could most definitely tell, I am rather new to SVG files as a whole and
have no such experience with them. Although I should soon be able to work
on the diff as soon as I have an opening in my schedule. Anyway I hope that
I don't make you have to work more for my faulty icon request and If I need
assistance I will make sure to do It when I don't have a resource available
to help me. Sorry for the extra trouble and I will be sure to come back
when I finish with the rework.

Thanks again for your help,
Adriano

File: icons/arduino-white.svg (hunk 1 above)
Comment (adrianoluengo) [reply icons/arduino-white.svg]: I don't quite know how to compound the paths in an SVG file. Do you have a certain site I could use to learn how to do this.

Comment (adrianoluengo) [reply icons/arduino-white.svg]: So I ended up using a different SVG file that wasn't as messy as the last one so you should notice some extra commits in the pull request in regards to replacing the file with a different one that has a black(subject to change) color. Once I fix that It should be set.

Comment (adrianoluengo) [reply icons/arduino-white.svg]: Just added the new logo, complete with paths, 24x24 size, and optimization. Just to note, the color is now white.

**Comment by adrianoluengo** [comment]
I may need to edit some elements since Travis-CI isn't successfully completing the build test. It seems that paths aren't compounded as you noticed, and now that the files is more easily read I can take a closer look into compounding the paths.

**Comment by ericcornelissen** [comment]
> I may need to edit some elements since Travis-CI isn't successfully completing the build test. It seems that paths aren't compounded as you noticed, and now that the files is more easily read I can take a closer look into compounding the paths.

That was not the only issue 😅 There where a couple of attributes on the SVGs `<path>` that [we prefer not to have](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#5-check-the-icon) and also, because you named the SVG file `arduino-logo.svg` our build system was not able to find it (the filename should correspond to the name in `simple-icons.json`).

I resolved these small issues for you so that the icon can be included in the package 👍 I hope you don't mind.

**Comment by ericcornelissen** [comment]
Also thanks for the reviews @hylickipiotr and @davidklebanoff

**Comment by adrianoluengo** [comment]
Yeah all I can say is thank you guys so much. I tried and failed to correct everything and I personally thank you guys for sorting it all out. I am completely new to SVG's and haven't actually played around with them ever and I hope that I'm the future I can do a bit more research to understand it all and properly work with SVG files.
Many thanks,
Adriano

Adriano Luengo

**Pull request merged**